examples/grpc.core/README.md

```diff
@@ -0,0 +1,17 @@
+# Grpc.Core Instrumentation for OpenTelemetry .NET - Examples
+
+Project structure:
+
+* Examples.GrpcCore.AspNetCore
+
+  This is an all-in-one sample that shows how to use the client and server
+  interceptors. The main project is new a vanilla webapi project created
+  using dotnet new ...
+
+  When launched, the project will hit the <http://localhost:5000/weather>
+  endpoint, make a call to a local in-process gRPC Core service running at
+  localhost:5001.
+
+  The end result is 3 spans dumped via the console exporter. A server span for

```

Comment:
would be good to clarify who collected the 3 spans and their parent/child relation.
The localhost:5000 server span is the root span I assume, and is collected by AspNetCore instrumentation.
Then then client span which is child of the above, and is collected by the grpc core instrumentation...and finally the server span which would be child of the above and collected by grpc core.

Reply:
I'll make some adjustments to the README files now.